test/parallel/test-buffer-tostring-binary.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
always need to include `require('../comon')`. It contains checks for after the test runs.